backend/settings_saml2.json

```diff
@@ -41,6 +41,7 @@
                 "want_name_id": true
             }
         },
+        "allow_unknown_attributes": true,
```

Comment:
@Millefeuille42 I asked another way: is there a reason we decide to refuse unknown attributes?

Reply:
I've not found any reason to refuse unknown attributes to be fair. I do think that this option is here to prevent the parsing + loading in memory of unused attributes (for optimization)

Reply:
ok.